### PR TITLE
Patch virtual classes regression bug, attribute comment parsing bug, initial config load bug, and add issue templates

### LINKED ISSUE #210: In the branch external-storage, the Declare complains about comment with some characters.
Here is the error:
```
Error using dj.internal.Declare.compileAttribute (line 348)
illegal characters in attribute comment "code version of the stimulus, usually two numbers "maze_version=2.1, code_version = 4.0". In the future: a githash?"
```
I tried removing the : as well, but still got the same error.

### LINKED ISSUE #335: Incorrect parsing of `attribute`'s type - incorrectly read from the comments
using `.getSchema()` on an existing table with the following definition:

```
@schema
class ProbeAdjustment(dj.Manual):
    definition = """
    -> ProbeInsertion
    adjustment_time: datetime
    ---
    -> [nullable] Session
    -> reference.Experimenter
    estimated_depth_change=null:    float  # (um) positive: moving more dorsal (shallower), negative: moving more ventral (deeper)
    adjustment :                    varchar(128)  # new depth (ideally)
    """
```

this error occurred:
```
Error using dj.internal.Declare.matchType (line 417)
UnsupportedType: Attribute type ' moving more dorsal (shallower), negative' is not a valid
type.
```

### LINKED ISSUE #336: Error in calling `getSchema()` for existing schema with `dj.Imported` (and likely `dj.Computed`) tables
Error message:

```
Error using dj.Imported
Abstract classes cannot be instantiated.  Class 'dj.Imported' inherits abstract methods or
properties but does not implement them.  See the list of methods and properties that
'dj.Imported' must implement if you do not intend the class to be abstract.

Error in dj.internal.TableAccessor (line 41)
                self.(name) = dj.(tierClass{2})(className{1});

Error in dj.Schema (line 69)
            self.v = dj.internal.TableAccessor(self);

Error in ephys.getSchema (line 5)
    schemaObject = dj.Schema(dj.conn, 'ephys', [db_prefix, 'ephys']);
```

### PR DESCRIPTION
**Pull request by guzman-raphael**
- Resolves #336 issue of instantiating classes with abstract methods (`dj.Imported`, `dj.Computed`) by using super class `dj.internal.UserRelation` and parsing table name to determine table type. This use case is triggered when using the virtual class operation. Added `dj.ERD.getTier` utility method.
- Resolves #210, #335 by properly escaping `"` and `/` characters in attribute comments.
- Resolves issue related to loading DataJoint config from file on initial MATLAB boot.
- Adds issue templates for bug reports and new feature requests.

This resolves the blocker that @ttngu207 reported related to Moser. Thinking to release `3.4.1` as introduced here and rename milestone from `3.4.1` -> `3.4.2` if others agree.

Fix #336
Fix #210
Fix #335

### CODE REVIEW

**Comment by ttngu207** [comment]
@guzman-raphael , I've installed 3.4.1 from your fork, tested the fix and here confirming that it has resolved issue #335 and #336.

**Comment by ttngu207** [comment]
@guzman-raphael , installed and tested 3.4.2, confirming here that issue #335 is resolved

File: +dj/Computed.m [hunk 1]
```diff
@@ -1,5 +1,10 @@
 classdef Computed < dj.internal.AutoPopulate
     % defines a computed table
+    properties(Constant)
+        tierRegexp = ['(?<computed>' ...
```
Comment (dimitri-yatsenko) [inline +dj/Computed.m:4]: I think `sprintf` would be clearer than these types of concatenations.

File: +dj/Imported.m [hunk 2]
```diff
@@ -1,5 +1,10 @@
 classdef Imported < dj.internal.AutoPopulate
     % defines an imported table
+    properties(Constant)
+        tierRegexp = ['(?<imported>' ...
```
Comment (dimitri-yatsenko) [inline +dj/Imported.m:4]: use `sprintf` for formatted strings.

**Review by dimitri-yatsenko — APPROVED** [review]
Very nice. I have reviewed and approve but I prefer to replace most string concatenations with sprintf for clearer formatting.

File: +dj/Computed.m (hunk 1 above)
Comment (guzman-raphael) [reply +dj/Computed.m]: @dimitri-yatsenko Updated styling per your feedback.

File: +dj/Imported.m (hunk 2 above)
Comment (guzman-raphael) [reply +dj/Imported.m]: Addressed.

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**